Regrtest summary

Ryex opened the pull request:
Add summaries for tests to quickly see passed | skipped | failed information

Screenshots:
![image](https://user-images.githubusercontent.com/508861/82963407-09f67b00-9f80-11ea-9482-b1c9d3e9499e.png)

This adds an `include` for a result summary: this `include` can also do a badge like SVG if `svg=true` is passed. but defaults to the table block seen above.

it can do *both* if `svg=true` and `table=true` are passed at the same time.

passing just `table=true` gets you just the table.

![image](https://user-images.githubusercontent.com/508861/82963734-dcf69800-9f80-11ea-8bba-3a0a57292027.png)

I slightly modified the template to allow this include to be added to the right of the head for test groups.

mireille-raad commented:
This is awesome and adds a lot of information to the page :D Also, neat way to add them!

I have some comments on styles/colors, but those are really minor details/up to the maintainer discretion.

I would ditch the "rust"/"orange" color that I originally used as a background for the header and maybe use black instead. black is more neutral and might go better with the green/yellow/red.

The css class is `results-test` in style.css.... on line 253. 

https://github.com/RustPython/rustpython.github.io/blob/c14ba36e3b4fb192fb497e51b8375d5d1e64e214/assets/style.css#L253-L259
 
(it is not the best class name now that I think of it ;p )

also for the bar height, maybe 20px instead of 40?

```
<div class="row" 
    style="background: linear-gradient(to right, 
            green {{ pass_percent }}%, 
              yellow {{ pass_percent }}% {{ skip_percent | plus: pass_percent }}%, 
              red {{ skip_percent | plus: pass_percent }}% {{ fail_percent | plus: pass_percent | plus: skip_percent }}%
            ); 
           height: 40px;">
```

Ryex commented:
I adjusted to colors a bit to contrast better with the background, also adjusted the bar height to 20px
and fixed the border color for the table

![image](https://user-images.githubusercontent.com/508861/82976957-b430cb80-9f9d-11ea-8866-782c425fe7f7.png)

coolreader18 commented:
Looks great!
